Formula/jfrog-cli.rb

```diff
@@ -31,6 +31,8 @@ def install
     # Install fish completion
     output = Utils.safe_popen_read(bin/"jf", "completion", "fish")
     (fish_completion/"jf.fish").write output
+
+    exec "#{bin}/jf intro"
```

Comment:
This will only run in CI, and even if you put it in a post install I don't think it fits with the way brew users expect installs to work.